admin-dev/themes/new-theme/.webpack/common.js

```diff
@@ -49,6 +49,8 @@ module.exports = {
     manufacturer_address: './js/pages/manufacturer/address',
     cms_page: './js/pages/cms-page',
     form_popover_error: './js/components/form/form-popover-error',
+    tax: './js/pages/tax',
```

Comment:
I don't get what tax comes to do in this PR?
If it was forgotten from another PR I'm ok with it, I'm mostly curious

Reply:
Hmmm, at some point there was `tax` entry duplication in `develop` branch, so I think it came from rebase, I'll remove it. :+1: